h2/src/test/org/h2/test/store/TestMVStore.java

```diff
@@ -1567,14 +1567,14 @@ private void testMeta() {
         try (MVStore s = openStore(fileName)) {
             s.setRetentionTime(Integer.MAX_VALUE);
             Map<String, String> m = s.getMetaMap();
-            assertEquals("[]", s.getMapNames().toString());
+            assertEquals("[restorePoints]", s.getMapNames().toString());
```

Comment:
MVStore is also used as a standalone multi-map key-value storage, such change in it is unacceptable. It will break some applications. MVStore shouldn't expose any internal maps.